core/conversion/converters/impl/unary.cpp

```diff
@@ -14,7 +14,7 @@ namespace {
           {"aten::" #unary "(Tensor self) -> Tensor",                           \
            [](ConversionCtx *ctx, const torch::jit::Node *n,                    \
               args &args) -> bool {                                             \
-             auto in = args[0].ITensor();                                       \
+             auto in = args[0].ITensorOrFreeze(ctx);                                       \
```

Comment:
nit: can you align the \'s, also this might be another case of if the tensor is static we should be doing the computation outright instead of adding a layer so we might not need this

Reply:
done